.ci/scripts/test-release.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 
 FLEET_SERVER_VERSION=${1:?"Fleet Server version is needed"}
 
-PLATFORM_FILES=(darwin-arm64.tar.gz darwin-x86_64.tar.gz linux-arm64.tar.gz linux-x86_64.tar.gz linux-x86.tar.gz windows-x86_64.zip windows-x86.zip)
+PLATFORM_FILES=(darwin-aarch64.tar.gz darwin-x86_64.tar.gz linux-arm64.tar.gz linux-x86_64.tar.gz linux-x86.tar.gz windows-x86_64.zip windows-x86.zip)
 
```

Comment:
@v1v I've made a change here to generate the artifact with the appropriate suffix (darwin-aarch64), would that impact anything downstream on your side? This should only be m1 related mac.

Reply:
Thanks for the heads up, I think it should work in https://github.com/elastic/fleet-server/pull/1234 , it uses the make release goal. Once it's merged the `daily release artifacts` generation will run in `beats-ci`, so if it's not working we will detect those errors earlier